crates/parameters/src/json_operations.rs

```diff
@@ -0,0 +1,74 @@
+use serde_json::Value;
+
+pub fn merge_json(own: &mut Value, other: &Value) {
```

Comment:
Is there a better name than `json_operations`? "Operations" does not give much insight.

Reply:
`json_utilities`? or `serde_json_utilities` naming is not a strong suit of mine.

Reply:
Since we are essentially adding features to a foreign type here, how about making an `Ext`-ension trait and using the naming conventions associated with that?

Reply:
Are we talking about this?
https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html
I'll take a look at this. Thanks for the suggestion 🙂

Reply:
Wait, knoellle and me discussed this again. The trait does not solve the point that I made. You don't need to introduce any trait. Instead we decided that we propose to name this file just `json.rs`.

Reply:
Renames, the extension trait also confused me a bit :D  let me know if something else should be done.